### PR TITLE
Add entryValueReversed && entryValueReversedAsync to RedissonScoredSo…

### PR DESCRIPTION
**Pull request by weiqiyiji**
With unit test passed

### CODE REVIEW

File: src/test/java/org/redisson/RedissonScoredSortedSetTest.java [hunk 1]
```diff
@@ -653,12 +653,32 @@ public void testScoredSortedSetEntryRange() {
...
+
+        set.add(0, "a");
+        set.add(1, "b");
+        set.add(2, "c");
+        set.add(3, "d");
+        set.add(4, "e");
+
+        Collection<ScoredEntry<String>> r = set.entryRangeReversed(1, true, 4, false, 1, 2);
+        Assert.assertEquals(2, r.size());
+        ScoredEntry<String>[] a = r.toArray(new ScoredEntry[0]);
```
Comment (mrniko) [inline src/test/java/org/redisson/RedissonScoredSortedSetTest.java]: You can use `readAll` method instead

**Comment by mrniko** [comment]
Thank you!

**Comment by weiqiyiji** [comment]
Hi, when will u release a new version with this commit?

**Comment by mrniko** [comment]
During next week

**Comment by weiqiyiji** [comment]
Hi, there, when will u release a new version with this commmit?

**Comment by mrniko** [comment]
@weiqiyiji Hi! Sorry for delay, soon :)

**Comment by weiqiyiji** [comment]
@mrniko ok, thank you!

**Pull request merged**